src/ostorlab/cli/ci_scan/run/run.py

```diff
@@ -0,0 +1,105 @@
+"""Module for the command run inside the group ci_scan.
+Example of usage:
+- ostorlab --api_key='myKey' ci_scan run --plan=free --break_on_risk_rating=medium --title=test_scan [asset] [options].
+"""
+import multiprocessing
+import click
+import time
+
+from ostorlab.cli.ci_scan.ci_scan import ci_scan
+from ostorlab.apis import scan_create as scan_create_api
+from ostorlab.apis import scan_info as scan_info_api
+from ostorlab.apis.runners import authenticated_runner
+from ostorlab.cli.ci_scan.run.ci_logger import console_logger, github_logger, logger
+
+MINUTE = 60
+WAIT_MINUTES = 30
+SLEEP_CHECKS = 10 # seconds
+
+RATINGS_ORDER = {
+    'high': 0,
+    'medium': 1,
+    'low': 2,
+    'potentially': 3
+}
+
+CI_LOGGER = {
+    'console': console_logger.Logger(),
```

Comment:
I would prefer only having a reference to the class and not creating a full instance that will not be used.